rdmo/management/management/commands/merge_attributes.py

```diff
@@ -0,0 +1,228 @@
+            model,
+            lookup_field,
+            field_introspection,
+            replacement_function,
+            source=source,
+            target=target,
+            save=save
+        )
+        results[model._meta.verbose_name_raw] = replaced_attributes
+
+    if save and all(a['saved'] for i in results.values() for a in i):
+        try:
+            source.delete()
+        except source.DoesNotExist:
+            pass
+    return results
+
+
+def main(source, target, save=False):
+    return replace_attribute_on_elements(source, target, REPLACE_ATTRIBUTE_MODEL_HELPERS, save)
+
+
+class Command(BaseCommand):
+    help = 'Replace an attribute with another attribute across multiple models'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--source',
+            required=True,
+            help='The URI of the source attribute that will be replaced by the target and will be deleted'
+        )
+        parser.add_argument(
+            '--target',
+            required=True,
+            help='The URI of the target attribute that will be used to replace the source'
+        )
+        parser.add_argument(
+            '--save',
+            action='store_true',
+            help='''If specified, the changes will be saved and the source attribute will be deleted.\
+                    If not specified, the command will do a dry run.'''
+        )
+
+    def handle(self, *args, **options):
+        source = options['source']
+        target = options['target']
+        save = options['save']
+        verbosity = options.get('verbosity', 1)
+
+        try:
+            source = get_valid_attribute(source, message_name='Source')
+            target = get_valid_attribute(target, message_name='Target')
+            results = main(source, target, save)
+
+            if verbosity >= 1:
+                if verbosity >= 2:
+                    affected_instances_msg = make_affected_instances_message(results)
+                    if affected_instances_msg:
+                        self.stdout.write(affected_instances_msg)
+                affect_elements_msg = make_elements_message(results)
+                affected_projects_msg = make_affected_projects_message(results)
+                if save:
+                    self.stdout.write(self.style.SUCCESS(
+                        f"Successfully replaced source attribute {source.uri} with {target.uri}.\n"
+                        f"Source attribute {source.uri} was deleted.\n"
+                        f"{affect_elements_msg}\n"
+                        f"{affected_projects_msg}"
+                    ))
+                else:
+                    self.stdout.write(self.style.SUCCESS(
+                        f"Source attribute {source.uri} can be replaced with {target.uri}.\n"
+                        f"{affect_elements_msg}\n"
+                        f"{affected_projects_msg}"
+                    ))
+        except ValueError as e:
```

Comment:
Is this meant to catch all ValueErrors in the whole code? Exceptions should be more specific to functions or a few lines of code.